src/qt/rpcconsole.cpp

```diff
@@ -1109,6 +1114,12 @@ void RPCConsole::updateDetailWidget()
         peerAddrDetails += "<br />" + tr("via %1").arg(QString::fromStdString(stats->nodeStats.addrLocal));
     ui->peerHeading->setText(peerAddrDetails);
     ui->peerServices->setText(GUIUtil::formatServicesStr(stats->nodeStats.nServices));
+    ui->peerRelayTxes->setText(stats->nodeStats.fRelayTxes ? "Yes" : "No");
+    QString bip152_hb_settings;
+    if (stats->nodeStats.m_bip152_highbandwidth_to) bip152_hb_settings += "To";
+    if (stats->nodeStats.m_bip152_highbandwidth_from) bip152_hb_settings += (bip152_hb_settings == "" ? "From" : "/From");
```

Comment:
nit: Consider using `.isEmpty()` instead of `== ""`, some times it might be costly:
https://wiki.qt.io/Using_QString_Effectively

Reply:
Slightly prefer isEmpty as well, but this shouldn't have any impact on performance and should be only a stylistic matter

Reply:
@MarcoFalke for std::string, a template class that has probably all code inlined - maybe (though `== ""` for `std::string` is also generating more code - see https://godbolt.org/z/h8r1Wq).

But QString invokes out-of-line`QString::compare_helper` with size set to `-1`, so it will, in the runtime, call `strlen` to determine lenght.. while `isEmpty()` is just inline `{ return d->size == 0; }`. 

Using `== ""` to check empty string bloats code for a else trivial check.

I agree that we don't need any (premature-)optimizations here (not performance critical code at all), but _not_ using `isEmpty()` (or std::string::empty() for that matter) is just a needless _pessimization_ and (relative) code bloat.

Reply:
Agree, thanks @Talkless, done in #221.